src/app/charts/core/files/secrets/tpl/arkcase-oidc.yaml

```diff
@@ -1,6 +1,14 @@
 {{- $oidc := (include "arkcase.oidc" $ | fromYaml) -}}
 {{- if $oidc }}
+  {{- if and (hasKey $oidc "legacyMode") (include "arkcase.toBoolean" $oidc.legacyMode) }}
+    {{- if (hasKey $oidc.clients "arkcase") }}
```

Comment:
I ended up in this way since `with` doesn't have an `else`.  Still cleaner than before, and fails when legacy mode is enabled without an arkcase key.

Reply:
actually `with` does have `else`... there was a defect elsewhere in my template.  Refactored again using `with / else`.